packages/smart-contracts/test/contracts/BatchConversionErc20Payments.test.ts

```diff
@@ -0,0 +1,586 @@
+
+      // update batch payment proxies, and batch fees
+      await testBatchConversionProxy.setPaymentErc20Proxy(erc20FeeProxy.address);
+      await testBatchConversionProxy.setPaymentEthProxy(ethereumFeeProxy.address);
+      await testBatchConversionProxy.setPaymentErc20ConversionProxy(
+        testErc20ConversionProxy.address,
+      );
+      await testBatchConversionProxy.setPaymentEthConversionProxy(testEthConversionProxy.address);
+
+      await testBatchConversionProxy.setBatchFee(batchFee);
+      await testBatchConversionProxy.setBatchConversionFee(batchConvFee);
+
+      // set ERC20 tokens
+      DAI_address = localERC20AlphaArtifact.getAddress(network.name);
+      testERC20 = new TestERC20__factory(signer1).attach(DAI_address);
+
+      FAU_address = secondLocalERC20AlphaArtifact.getAddress(network.name);
+      testERC20b = new TestERC20__factory(signer1).attach(FAU_address);
+      batchAddress = testBatchConversionProxy.address;
+
+      setBatchConvFunction(useBatchRouter, signer1);
+    });
+
+    beforeEach(async () => {
+      fromDiffBalanceExpected = BigNumber.from(0);
+      toDiffBalanceExpected = BigNumber.from(0);
+      feeDiffBalanceExpected = BigNumber.from(0);
+      await testERC20.approve(testBatchConversionProxy.address, thousandWith18Decimal, {
+        from,
+      });
+      await testERC20b.approve(testBatchConversionProxy.address, thousandWith18Decimal, {
+        from,
+      });
+      // get balances of testERC20 token
+      fromOldBalance = await testERC20.balanceOf(from);
+      toOldBalance = await testERC20.balanceOf(to);
+      feeOldBalance = await testERC20.balanceOf(feeAddress);
+
+      // create a default convDetail
+      path = [USD_hash, DAI_address];
+      getConvToPayAndConvDetail(to, path, amountInFiat, feesAmountInFiat, 0, chainlinkPath);
+    });
+
+    afterEach(async () => {
+      // check balances of testERC20 token
+      checkBalancesForOneToken(
+        testERC20,
+        fromOldBalance,
+        toOldBalance,
+        feeOldBalance,
+        fromDiffBalanceExpected,
+        toDiffBalanceExpected,
+        feeDiffBalanceExpected,
+      );
+    });
+
+    after(async () => {
+      // restore previous values for consistency
+      await testBatchConversionProxy.setBatchFee(30);
+      await testBatchConversionProxy.setBatchConversionFee(30);
+    });
+
+    describe(useBatchRouter ? 'Through batchRouter' : 'Without batchRouter', () => {
+      describe('batchERC20ConversionPaymentsMultiTokens with DAI', async () => {
+        it('allows to transfer DAI tokens for USD payment', async () => {
+          await onePaymentBatchConv(path);
+        });
+        it('allows to transfer DAI tokens for EUR payment', async () => {
+          path = [EUR_hash, USD_hash, DAI_address];
+          await onePaymentBatchConv(path);
+        });
+        it('allows to transfer 2 transactions DAI tokens for USD payment', async function () {
```

Comment:
I don't understand the title of this unit test and the absence of assertion looks weird.